index.js

```diff
@@ -0,0 +1,7 @@
+document.getElementById("myCanvas");
```

Comment:
Is this meant to be assigned to a variable?

Reply:
yes but it's not done yet in this stage so it was just a placeholder